docs/rfcs/XeGPU.md

```diff
@@ -22,8 +22,8 @@ Below is a summary.
 |store_nd	| operation ::= XeGPU.store_nd $value, $tdesc attr-dict : type($value), type($tdesc) | XeGPU.store_nd %value, %tdesc2 {L1_hint = uncached, L3_hint = uncached} : vector<8x16xbf16>, tensor_desc<8x16xbf16> |
 |update_nd_offset	| operation ::= XeGPU.update_nd_offset $tdesc, $delta0, $delta1 : type($tdesc), index, index -> type($tdesc)	| %tdesc_updated = XeGpu.update_nd_offset %tdesc, %offset_x, offset_y, tensor_desc<8x16xbf16>, index, index -> tensor_desc<8x16xbf16> |
 |prefetch_nd	| operation ::= XeGPU.prefetch_nd $tdesc, attr-dict : type($tdesc) | XeGPU.prefetch_nd %tdesc2: tensor_desc<8x16xbf16> |
-|alloc_nbarrier	| operation ::= XeGPU.alloc_nbarrier $barrier_couter : uint8_t | XeGPU.alloc_nbarrier %nbarrier_count: Uint8_t |
-|create_nbarrier	| operation ::= XeGPU.create_nbarrier $nbarrier_id, $nbarrier_role attr-dict : uint8_t, type($nbarrier_role) -> type($nbarrier) | %nbarrier = XeGPU.create_nbarrier %nbarrier_id, %nbarrier_role {num_producers = 2, num_consumers = 2} : Uint8_t, nbarrier_role -> !XeGPU.nbarrier |
+|create_nbarrier	| operation ::= XeGPU.create_nbarrier $barrier_num : index | XeGPU.creat_nbarrier %nbarrier_num: Uint8_t |
```

Comment:
I am thinking of whether we need to add return value for `create_nbarrier` with type of vector<#num x index>, otherwise, $nbarrier_id in init_nbarrier looks wired.

Reply:
Here create_nbarrier should really renamed to alloc_nbarrier.  

Previously create_nbarrier and alloc_nbarrier is confusing, so I did the change and seems introduce a different confusion.

How about use alloc_nbarrier and init_nbarrier?